Gemfile

```diff
@@ -4,3 +4,5 @@ gem 'minitest'
 gem 'rubocop', '0.36.0'
 gem 'simplecov'
 gem 'rake'
+gem 'values'
```

Comment:
Good point.

It depends on where you want to draw the line, the 2 I added were convenient for adding the functionality I wanted. I'll look into what it would take to eliminate the dependencies. I suspect not much.